README.md

```diff
@@ -54,16 +54,17 @@ And then update your dependency to the current SNAPSHOT version.
 
 Configuration Options
 ---------------------
-* port_number - the port number that the nodes will communicate over.  This needs to be the same on all nodes.  The default is 7800.
-* port_range - the number of additional ports to be probed for membership. A port_range of 0 does not probe additional ports. Example: initial_hosts=A[7800] port_range=0 probes A:7800, port_range=1 probes A:7800 and A:7801.  The default is 50.
-* tags - A comma delimited list of EC2 node tag names.  The current nodes values are matched against other nodes to find
+* `port_number` - the port number that the nodes will communicate over.  This needs to be the same on all nodes.  The default is 7800.
+* `port_range` - the number of additional ports to be probed for membership. A port_range of 0 does not probe additional ports. Example: initial_hosts=A[7800] port_range=0 probes A:7800, port_range=1 probes A:7800 and A:7801.  The default is 50.
+* `tags` - A comma delimited list of EC2 node tag names.  The current nodes values are matched against other nodes to find
```

Comment:
There is obviously broken or undefined behavior here, when tags are not defined.  This could easily lead to surprising results.